### ISSUE TITLE
Extension settings windows don't stay open

### ISSUE THREAD

**Issue by distefam** [paperwm/PaperWM#771]
**Describe the bug**
Extension settings closes without user intervention and cannot be opened again.

**To Reproduce**
1. Open Extensions app
2. Click on gear next to PaperWM extension
3. Click around inside the settings (change tabs, for example)
4. Observe that PaperWM settings closes/crashes
5. Try to open PaperWM settings again
6. Observe that settings will not open

**Expected behavior**
Settings will stay open

**Screenshots**

**System information:**

```
Distribution: Fedora Linux
GNOME Shell 45.3
Display server: Wayland
PaperWM branch/tag: release
PaperWM commit: 35afc1a117650dbd9b7ee2afccc691e0b4514e89
Enabled extensions:
- dash-to-dock@micxgx.gmail.com
- clipboard-indicator@tudmotu.com
- espresso@coadmunkee.github.com
- nightthemeswitcher@romainvigier.fr
- appindicatorsupport@rgcjonas.gmail.com
- lockkeys@vaina.lt
- do-not-disturb-while-screen-sharing-or-recording@marcinjahn.com
- paperwm@paperwm.github.com
- grand-theft-focus@zalckos.github.com
- search-light@icedman.github.com
- notification-position@drugo.dev
- Move_Clock@rmy.pobox.com
- display-brightness-ddcutil@themightydeity.github.com
- launch-new-instance@gnome-shell-extensions.gcampax.github.com
- mousefollowsfocus@matthes.biz
```

**Additional context**
This applies to all extension settings, not just PaperWM. I'm not sure it's a PaperWM bug or something to do with Gnome shell or the Extensions app.

**Comment by Lythenas** [comment]
Can you check the logs (e.g. `journalctl -ef`) if this is the same issue as #749?

**Comment by jtaala** [comment]
Hey @distefam, this was not a PaperWM specific (or caused) issue.  It was a recent gjs issue that has been fixed.  I submitted the gjs issue upstream, see

https://gitlab.gnome.org/GNOME/gjs/-/issues/598

The affected version of GJS was 1.78.2.  Check your gjs version from terminal and let us know what it is:

```
gjs --version
```

The fixed version is `1.78.3`.  Definitely update your distro packages.
